test/fixtures/permission/fs-read.js

```diff
@@ -265,6 +265,13 @@ const regularFile = __filename;
     permission: 'FileSystemRead',
     resource: path.toNamespacedPath(blockedFile),
   }));
+  assert.throws(() => {
+    fs.watch(blockedFileURL, () => {});
+  }, common.expectsError({
```

Comment:
FWIW the `expectsError` is unnecessary
```suggestion
  }, {
```